src/ripple/app/tests/Offer.test.cpp

```diff
@@ -30,6 +30,21 @@ namespace test {
 
 class Offer_test : public beast::unit_test::suite
 {
+    XRPAmount reserve(jtx::Env& env, std::uint32_t count = 0)
+    {
+        return env.current()->fees().accountReserve (count);
+    }
+
+    XRPAmount reserve(jtx::Env& env, jtx::Account const& account)
```

Comment:
Just be mindful that `jtx::Account` constructors are designed to support implicit conversions. If `Account` ever becomes constructible from `std::uint32_t`, this will break.